slick.grid.js

```diff
@@ -800,10 +800,17 @@ if (typeof Slick === "undefined") {
     // Given a column object, return a jquery element with HTML for the column
     // Can be overridden by providing a function to options.columnHeaderRenderer
     function columnHeaderRenderer(column) {
-      var $el = $("<div class='cell' />")
-        .html("<span class='name'>" + column.name + "</span>")
-        .attr("title", column.toolTip || "");
-      return $el;
+
+      // 50% faster using native API, versus the old jQuery way
+      // jQuery's .html() is particularly slow
+      //
+      // Saves roughly 0.5ms * N in synchronous processing time, where N is the number of columns
+      var d = document.createElement('div')
+      d.className = 'cell';
+      d.innerHTML = "<span class='name'>" + column.name + "</span>";
+      d.title = column.toolTip || "";

```

Comment:
I don't know if setting the title attr is expensive, but do you want it to be a blank string when falsey? I'd lean towards `if (column.toolTip) { d.title = column.toolTip }